### ISSUE TITLE
[FEATURE REQUEST] Add capability to save species database to file in run directory for input in GCPy 

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#109]
Currently GCPy uses a species_database.json file to obtain species information (e.g. molecular weight) for use in generating emissions and mass tables for benchmarking. Instead of having two files containing this information (Headers/species_database_mod.F90 in GC and core/species_database.py in GCPy), it would be better to have GEOS-Chem save out this information to a file in the run directory that can be passed as input in GCPy. This would ensure you have the information for all species that you are trying to process in GCPy.

**Comment by msulprizio** [comment]
This can be done simultaneously with https://github.com/geoschem/geos-chem/issues/110

**Comment by msulprizio** [comment]
@yantosca wrote:

>There is code to save out the species database from GC but it's commented out. Look for this in init_species_database, and uncomment those lines 

```
    !-----------------------------------------------------------------------
    ! Print Species Database to JSON format
    ! NOTE: Comment this out unless you really need it!
    !CALL SpcData_To_JSON( am_I_Root, SpcData, RC )
    !STOP
```

>If you run that in a benchmark simulation that should get all the species

**Comment by msulprizio** [comment]
Resolved in 7a169f7e2ffc69927f1f512c9550f59302999633
